macros/anaUltraThreshold.py

```diff
@@ -178,18 +283,42 @@
         pass
     pass
 outF.Close()
-txt = open(filename+"/vfatConfig.txt", 'w')
+txt_vfat = open(filename+"/vfatConfig.txt", 'w')
 
 print "trimRange:"
 print trimRange
 print "vt1:"
 print vt1
 
-txt.write("vfatN/I:vt1/I:vt1new/I:trimRange/I\n")
-for i in range(0,24):
-    txt.write('%i\t%i\t%i\t%i\n'%(i, vt1[i], vt1new[i], trimRange[i]))
+txt_vfat.write("vfatN/I:vt1/I:trimRange/I\n")
```

Comment:
Yeah some mistake on my part when forking, my bad...for sure will do better job of sync'ing in future.

In any event I think lines 134-139 are in the spirt of what we are looking for.